mreg/authentication.py

```diff
@@ -20,7 +27,11 @@ def authenticate_credentials(self, key):
         if not token.user.is_active:
             raise exceptions.AuthenticationFailed('User inactive or deleted')
 
-        if token.created < timezone.now() - timedelta(hours=EXPIRE_HOURS):
+        if token.last_used < timezone.now() - timedelta(hours=EXPIRE_HOURS):
             raise exceptions.AuthenticationFailed('Token has expired')
 
+        # Update the created time of the token to keep it valid.
+        token.last_used = timezone.now()
```

Comment:
This is automatically done with a `auto_now=True` `DateTimeField`.
Keep to comment, but add that it doesn't need to explicitly update the field. This is similar to the `BaseModel`'s `updated_at` field.

Reply:
Using `auto_now` makes a lot of sense.  I had to mock the current time in order to make the `test_force_expire` test pass because we no longer control the `last_used` field, and added a development dependency on `mock`.